### ISSUE TITLE
Theme options broken

### ISSUE THREAD

**Issue by loopy750** [uso-archive/data#51]
After being mirrored from Userstyles, comments are being removed, including theme comments, eg: `/*[[font]]*/`

Changing settings on the theme therefore has no effect.

**Comment by tophf** [comment]
Apparently they've disabled the API to get the original CSS just to thwart our efforts. They even broke "Show CSS" button on their site.

**Comment by loopy750** [comment]
Dang, thanks for the update. They really broke a lot of stuff with the major overhaul, including browsing with Firefox, as mentioned in the other thread. I even reported the issue, and am not one for a conspiracy, but it sure seems intentional.

Good to know I didn't do something wrong, but sucks that it means it's broken for everyone.

**Comment by tophf** [comment]
@33KK, here's an idea: make the API request for style sections using unique random values for style variables, then replace them in the result with `/*[[name]]*/` comments.

**Comment by 33KK** [comment]
> @33KK, here's an idea: make the API request for style sections using unique random values for style variables, then replace them in the result with `/*[[name]]*/` comments.

That's exactly what I am doing right now. Dropdowns are indeed broken though. Unless they changed anything else the other settings should work

**Comment by 33KK** [comment]
Right, your styles only use dropdowns. That's why they are broken. Recovering dropdown options isn't trivial and I gave up on it because their website was (maybe still is) broken and didn't even parse sections correctly

**Comment by tophf** [comment]
The web site or do you use the API for getStyleCustomOptions?

**Comment by 33KK** [comment]
I do what their site is doing

**Comment by tophf** [comment]
Then maybe there's another way? I've been using getStyleCustomOptions and it still works.

**Comment by 33KK** [comment]
Looked at this again and again decided that I really don't want to deal with it 💀 

Basically what we have are dropdown keys and values and can get the css with the placeholders already replaced. It would be reasonably to just replace the value to the placeholder back, except the value can also exist in other places and wrong things will get replaced with the placeholder and then there's nested placeholders which is also fun to deal with I guess. I was thinking of doing some kind of diff thing to figure out what is actually changing but idk how to make it work exactly
